### ISSUE TITLE
Dropout in Keras VS Dropout in Hinton paper

### ISSUE THREAD

**Issue by MordFee** [fchollet/keras#1715]
Hello fchollet, or whoever is responsible of Keras.io website,

I have noted an misunderstanding between the literature and the Dropout feature in Keras package.

In Hinton paper, the best way to prevent overfitting with dropout is for dropout = 0.8 which means that they keep 80% of the input layer
In Keras, using dropout = 0.8 means that you set to zero 80% of the layer (cf here http://keras.io/layers/core/#dropout)

If you want to keep it this way, which I think make more sense than the paper, maybe you should emphasize it a little more in your website.

Best,
